tests/bench_dyncall.c

```diff
@@ -0,0 +1,381 @@
+#include <emscripten.h>
```

Comment:
In test_core.py this seems to be called `test_dyncalls.c`?   Not sure how this is working.

Reply:
This is a separate benchmark file. It is now out of date after some revision, I'll update this to latest when I get a chance, to make sure it tests the latest form of the API.